modules/gdscript/gdscript_warning.cpp

```diff
@@ -64,7 +64,10 @@ String GDScriptWarning::get_message() const {
 			return vformat(R"(The local %s "%s" is shadowing an already-declared %s at line %s.)", symbols[0], symbols[1], symbols[2], symbols[3]);
```

Comment:
Perhaps this makes sense for clarification and consistency?

```suggestion
			return vformat(R"(The local %s "%s" is shadowing an already-declared %s at line %s in the current class.)", symbols[0], symbols[1], symbols[2], symbols[3]);
```

Reply:
I will have to update current tests to reflect this update, is this okay to be done?

Reply:
Yes, otherwise the tests will fail.